src/core/pointcloud/qgslazinfo.cpp

```diff
@@ -302,6 +303,7 @@ QgsLazInfo QgsLazInfo::fromUrl( QUrl &url )
   // Fetch header data
   {
     QNetworkRequest nr( url );
+    QgsSetRequestInitiatorClass( nr, QStringLiteral( "QgsLazInfo" ) );
```

Comment:
This could also be applied to the VLR request which is following

Reply:
Good point, added now in [87b4590](https://github.com/qgis/QGIS/pull/52652/commits/87b45909f7c261a93f0d887d5d60d6103af86b68)